scripts/portstat_errors

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/python
+
+#####################################################################
+#
+# portstat_errors is a tool for summarizing network error statistics.
```

Comment:
It seems like creating a new `portstat_errors` utility causes a lot of code duplication. Can you instead modify the existing `portstat` utility to accept an optional flag which will cause it to output only errors?

Reply:
Fixed this to use existing portstat.